docs/UPLOADING_DOCUMENTS.md

```diff
@@ -0,0 +1,85 @@
+# Keeping the documents updated
+
+As our webpage is not connected to any database, the data provided are static and
+need to be updated manually.
+
+### The current workflow is:
+
+-   write the documentation in the [LaTeX](https://www.latex-project.org/) and
+    compile the files to the PDF
+-   add the documents into webpage static data
+-   rebuild the project
+-   upload the documents and the builded files to the webserver
+
+Each of these steps are described below in detail.
+
+## Writing the documentation
```

Comment:
better is "Writing the document" because we will write all the documents in Latex ("zápisnice, metodiky, retrospektivy")

Reply:
Fixed, replaced all occurences of "documentation" with "document".